apps/api/src/organization/controllers/region.controller.ts

```diff
@@ -40,30 +60,131 @@ export class RegionController {
     description: 'List of all regions',
     type: [RegionDto],
   })
-  @ApiQuery({
-    name: 'includeShared',
-    required: false,
-    type: Boolean,
-    description: 'Include shared regions',
+  async listRegions(@AuthContext() authContext: OrganizationAuthContext): Promise<RegionDto[]> {
+    const customRegions = await this.regionService.findAllByOrganization(authContext.organizationId, RegionType.CUSTOM)
```

Comment:
Have you considered reducing the number of db calls here?

We can create a dedicated service method for this that could `JOIN` or `OR` these things